frontend/src/util/config.tsx

```diff
@@ -1,5 +1,4 @@
 
-const host = `http://${window.location.hostname}:6969`;
-// const host = `http://localhost:6969`;
+const host = `http://${window.location.hostname}:${window.location.port}`;
```

Comment:
Is there any thing like window.location.host that combined both hostname and port?